platforms/minigame/platforms/taobao-mini-game/wrapper/builtin/WebSocket.js

```diff
@@ -55,10 +55,26 @@ export default class WebSocket {
     my.onSocketOpen(this._onOpen)
 
     this._onMessage = (res) => {
-      if (res && res.data && res.isBuffer) {
-        res.data = _utils.base64ToArrayBuffer(res.data)
+      function deepCopy(obj) {
+        if (typeof obj !== 'object' || obj === null) {
+          return obj;
+        }
+      
+        const copiedObj = {};
+      
+        for (const key in obj) {
+          if (obj.hasOwnProperty(key)) {
+            copiedObj[key] = deepCopy(obj[key]);
+          }
+        }
+      
+        return copiedObj;
+      }
+      let tmp = deepCopy(res);
+      if (tmp && tmp.data && tmp.isBuffer) {
+        tmp.data = _utils.base64ToArrayBuffer(tmp.data)
```

Comment:
So taobao platform only supports `string` response content?

Reply:
According to Taobao's official requirements, ArrayBuffers must be encoded in Base64 format.